torch/_inductor/compile_fx.py

```diff
@@ -400,11 +400,4 @@ def bw_compiler(model: torch.fx.GraphModule, example_inputs):
             partition_fn=functools.partial(
                 min_cut_rematerialization_partition, compiler="inductor"
             ),
-            # A "tiny" graph can actually decompose into multiple
-            # operators (if it's a decomposition) and inductor can
-            # do a better job on it in this case
-            #
-            # Also, for some reason, test_comprehensive___rmatmul___cpu
-            # fails without forcing a compile lol.
```

Comment:
I was going to ask why we’re removing this config, but the comment gives a pretty good reason (since dynamo doesn’t decompose, we don’t really know if the current graph is “small”)